android/src/main/java/com/onegini/mobile/sdk/reactnative/clean/use_cases/GetAuthenticatedUserProfileUseCase.kt

```diff
@@ -2,12 +2,16 @@ package com.onegini.mobile.sdk.reactnative.clean.use_cases
 
 import com.facebook.react.bridge.Promise
 import com.onegini.mobile.sdk.reactnative.OneginiSDK
+import com.onegini.mobile.sdk.reactnative.exception.OneginiWrapperErrors
 import com.onegini.mobile.sdk.reactnative.mapers.UserProfileMapper
 
 class GetAuthenticatedUserProfileUseCase(private val oneginiSDK: OneginiSDK) {
 
     operator fun invoke(promise: Promise) {
-        val profile = oneginiSDK.oneginiClient.userClient.authenticatedUserProfile
-        promise.resolve(UserProfileMapper.toWritableMap(profile))
+        oneginiSDK.oneginiClient.userClient.authenticatedUserProfile?.let { userProfile ->
+            promise.resolve(UserProfileMapper.toWritableMap(userProfile))
+            return;
+        }
+        promise.reject(OneginiWrapperErrors.NO_PROFILE_AUTHENTICATED.code, OneginiWrapperErrors.NO_PROFILE_AUTHENTICATED.message)
```

Comment:
I see that there's a `return` in `?.let` body but anyway I think it would be better for the future if you'd call  
` promise.reject(OneginiWrapperErrors.NO_PROFILE_AUTHENTICATED.code, OneginiWrapperErrors.NO_PROFILE_AUTHENTICATED.message)` after  `?:`elvis operator.